src/main/java/gregtech/integration/theoneprobe/provider/ActiveTransformerInfoProvider.java

```diff
@@ -0,0 +1,55 @@
+package gregtech.integration.theoneprobe.provider;
+
+import gregtech.api.GTValues;
+import gregtech.api.capability.GregtechCapabilities;
+import gregtech.api.capability.IMultiblockController;
+import gregtech.api.util.TextComponentUtil;
+import gregtech.api.util.TextFormattingUtil;
+import gregtech.common.metatileentities.multi.electric.MetaTileEntityActiveTransformer;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.common.capabilities.Capability;
+
+import mcjty.theoneprobe.api.IProbeHitData;
+import mcjty.theoneprobe.api.IProbeInfo;
+import mcjty.theoneprobe.api.NumberFormat;
+import mcjty.theoneprobe.apiimpl.elements.ElementProgress;
+import org.jetbrains.annotations.NotNull;
+
+public class ActiveTransformerInfoProvider extends CapabilityInfoProvider<IMultiblockController> {
+
+    @Override
+    public String getID() {
+        return GTValues.MODID + ":active_transformer_info_provider";
+    }
+
+    @Override
+    protected boolean allowDisplaying(@NotNull IMultiblockController capability) {
+        return capability instanceof MetaTileEntityActiveTransformer;
+    }
+
+    @Override
+    protected @NotNull Capability<IMultiblockController> getCapability() {
+        return GregtechCapabilities.CAPABILITY_MULTIBLOCK_CONTROLLER;
+    }
+
+    @Override
+    protected void addProbeInfo(IMultiblockController capability, IProbeInfo probeInfo, EntityPlayer player,
+                                TileEntity tileEntity, IProbeHitData data) {
+        if (capability.isStructureFormed() && capability instanceof MetaTileEntityActiveTransformer activeTransformer &&
```

Comment:
You don't need to check for if the capability is an instance of the ActiveTransformer here, since you override `allowDisplaying`. Having both is redundant, removing the override allows keeping the local variable creation though, to check if the AT is active.